docs/integrations/sources/stripe.md

```diff
@@ -26,6 +26,7 @@ This Source is capable of syncing the following core Streams:
 * [Subscriptions](https://stripe.com/docs/api/subscriptions/list) \(Incremental\)
 * [Subscription Items](https://stripe.com/docs/api/subscription_items/list)
 * [Transfers](https://stripe.com/docs/api/transfers/list) \(Incremental\)
```

Comment:
could you insert this in alphabetical order?

Reply:
Sure, done.